web/src/features/charts/ChartTitle.tsx

```diff
@@ -45,7 +48,26 @@ export function ChartTitle({
           </MoreOptionsDropdown>
         )}
       </div>
-      {unit && <div className="text-sm dark:text-neutral-300">{unit}</div>}
+      <div className="flex flex-row items-center justify-between">
+        {subtitle}
+        {unit && (
+          <div className="ml-auto text-xs text-[#8b8b8b] dark:text-[#848484]">{unit}</div>
+        )}
+      </div>
     </div>
   );
 }
+
+interface ChartSubtitleProps {
+  datetimes: Date[];
+  timeRange: TimeRange;
+}
+
+export function ChartSubtitle({ datetimes, timeRange }: ChartSubtitleProps) {
+  const { i18n } = useTranslation();
+  return (
+    <p className="text-xs text-neutral-600 dark:text-neutral-300">
+      {getDateRange(i18n.language, datetimes, timeRange)}
```

Comment:
```suggestion
      {getDateRange(i18n.languages[0], datetimes, timeRange)}
```
Language is the resolved language but languages[0] is the used language. They don't always match if the resolved language don't exists as it falls back to english.

Reply:
Ahhh whoops, good catch!